pkg/test/framework/components/echo/kube/deployment.go

```diff
@@ -415,7 +422,7 @@ spec:
           name: istio-vm-bootstrap
         {{- range $name, $value := $subset.Annotations }}
         {{- if eq $name.Name "sidecar.istio.io/bootstrapOverride" }}
-        - mountPath: /etc/istio/custom-bootstrap
+        - mountPath: /etc/istio-custom-bootstrap
```

Comment:
just curious: what drives this change?

Reply:
A fair question. Before, we had the `/etc/istio/config/mesh` file in the original docker image; now its added at runtime. But there is some conflict with the mount that behaves differently for these. I don't fully understand why, TBH, but this purely impacts tests with our fake VMs so I wasn't too worried